var/spack/repos/builtin/packages/hiop/package.py

```diff
@@ -274,37 +272,48 @@ def cmake_args(self):
     #
     # export SPACK_USER_CACHE_PATH=/tmp/spack
     # export SPACK_DISABLE_LOCAL_CONFIG=true
-    def test(self):
+
+    def test_N1pMDsEx1(self):
+        """Test N1pMDsEx1"""
         if not self.spec.satisfies("@develop") or not os.path.isdir(self.prefix.bin):
```

Comment:
```suggestion
        if not self.spec.satisfies("@develop") or not os.path.isdir(self.prefix.bin):
```

https://github.com/LLNL/hiop/tree/v0.6.1/src/Drivers/MDS - 0.6.1 seems to be the first version where this specific test existed.